extendr-api/src/wrapper/matrix.rs

```diff
@@ -361,41 +361,65 @@ impl<T, D> Deref for RArray<T, D> {
     }
 }
 
-#[test]
-fn matrix_ops() {
-    test! {
-        let vector = RColumn::new_column(3, |r| [1., 2., 3.][r]);
-        let robj = r!(vector);
-        assert_eq!(robj.is_vector(), true);
-        assert_eq!(robj.nrows(), 3);
-
-        let vector2 : RColumn<f64> = robj.as_column().ok_or("expected array")?;
-        assert_eq!(vector2.data().len(), 3);
-        assert_eq!(vector2.nrows(), 3);
-
-        let matrix = RMatrix::new_matrix(3, 2, |r, c| [
-            [1., 2., 3.],
-            [4., 5., 6.]][c][r]);
-        let robj = r!(matrix);
-        assert_eq!(robj.is_matrix(), true);
-        assert_eq!(robj.nrows(), 3);
-        assert_eq!(robj.ncols(), 2);
-        let matrix2 : RMatrix<f64> = robj.as_matrix().ok_or("expected matrix")?;
-        assert_eq!(matrix2.data().len(), 6);
-        assert_eq!(matrix2.nrows(), 3);
-        assert_eq!(matrix2.ncols(), 2);
-
-        let array = RMatrix3D::new_matrix3d(2, 2, 2, |r, c, m| [
-            [[1., 2.],  [3., 4.]],
-            [[5.,  6.], [7., 8.]]][m][c][r]);
-        let robj = r!(array);
-        assert_eq!(robj.is_array(), true);
-        assert_eq!(robj.nrows(), 2);
-        assert_eq!(robj.ncols(), 2);
-        let array2 : RMatrix3D<f64> = robj.as_matrix3d().ok_or("expected matrix3d")?;
-        assert_eq!(array2.data().len(), 8);
-        assert_eq!(array2.nrows(), 2);
-        assert_eq!(array2.ncols(), 2);
-        assert_eq!(array2.nsub(), 2);
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn matrix_ops() {
+        test! {
+            let vector = RColumn::new_column(3, |r| [1., 2., 3.][r]);
+            let robj = r!(vector);
+            assert_eq!(robj.is_vector(), true);
+            assert_eq!(robj.nrows(), 3);
```

Comment:
can we add comments above the assertion sections describing what is being asserted?